### PR TITLE
chore(networking): remove PutRecord kad query tracking.

### PR DESCRIPTION
**Pull request by joshuef**
This doesn't tell us anything useful and is more state to keep track of in the swarm driver

### Summary generated by Reviewpad on 20 Jul 23 12:35 UTC
This pull request removes the PutRecord kad query tracking in the networking module. This change eliminates unnecessary state and reduces complexity in the swarm driver.

### CODE REVIEW

File: sn_cli/src/subcommands/register.rs [hunk 1]
```diff
@@ -43,15 +47,40 @@ pub(crate) async fn register_cmds(cmds: RegisterCmds, client: &Client) -> Result
...
     let _register = ClientRegister::create_public_online(client.clone(), xorname, tag).await?;
-    println!("Successfully created register '{name}' at {xorname:?}, {tag}!");
-    Ok(())
+
+    let mut verification_attempts = 0;
+
+    while verification_attempts < VERIFICATION_ATTEMPTS {
+        match client.get_register(xorname, tag).await {
+            Ok(_) => {
+                println!("Successfully created register '{name}' at {xorname:?}, {tag}!");
```
Comment (grumbach) [inline sn_cli/src/subcommands/register.rs]: Being a bit pedantic here but `Ok(_)` here could well be another register created concurrently, we might want to check if it matches ours.

**Review by grumbach — COMMENTED** [review]
LGTM

File: sn_cli/src/subcommands/register.rs (hunk 1 above)
Comment (joshuef) [reply sn_cli/src/subcommands/register.rs]: Hmm, 
```
                if our_register.name() == returned_register.name();
```
is enough? comparing contents  might fail if we get _newer_ ?

Comment (joshuef) [reply sn_cli/src/subcommands/register.rs]: no, that'd be the same whatever for that get.... hmm. if we're doing this we need to tolerate potential _new_ entries

Comment (joshuef) [reply sn_cli/src/subcommands/register.rs]: Okay, i think I have a good enough way forwards here. Just checking the unferlying and if not matching erroring so the user can take action.

**Pull request closed without merging**